server/jamovi/server/compute/functions.py

```diff
@@ -291,12 +291,12 @@ def VBOXCOXLAMBDA(values: float):
 
 @row_wise
 def BOXCOX(index, x: float, lmbda: float = VBOXCOXLAMBDA):
-    if lmbda > 0:
-        return (x ** lmbda - 1) / lmbda
+    if x < 0:
+        return NaN
     elif lmbda == 0:
```

Comment:
You might also need to catch the special case where x == 0 and lmbda < 0, because now you would get a `ZeroDivisionError` I think. I think it woulf need to return a `-inf` in that case, see https://docs.scipy.org/doc/scipy/reference/generated/scipy.special.boxcox.html#scipy.special.boxcox